Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -466,12 +466,26 @@ public struct ObjcLibrary: BazelTarget, UserConfigurable, SourceExcludable {
                 return target is ObjcLibrary
             }.map { ($0 + "_hdrs").toSkylark() }
         }
-       
-        let podSupportHeaders = GlobNode(include: AttrSet<Set<String>>(basic: [PodSupportSystemPublicHeaderDir + "**/*"]),
-                                                         exclude: AttrSet<Set<String>>.empty).toSkylark()
+
+        let podSupportHeadersGlob = Set([PodSupportSystemPublicHeaderDir + "**/*"])
+        let podSupportHeaders = GlobNode(
+                include: AttrSet<Set<String>>(basic: podSupportHeadersGlob),
+                exclude: AttrSet<Set<String>>.empty
+        ).toSkylark()
+
+        // We need to explicitly exclude the pod support headers in case the headers include a bare `**/*` which would
+        // cause the pod support headers to be doubly included.
+        let headersWithoutPodSupport = GlobNode(
+                include: headers.include,
+                exclude: AttrSet(
+                        basic: (headers.exclude.basic ?? Set()).union(podSupportHeadersGlob),
+                        multi: headers.exclude.multi
+                )
+        )
+
         if lib.isTopLevelTarget {
             var exposedHeaders: SkylarkNode = podSupportHeaders .+.
-                headers.toSkylark() .+. depHdrs.toSkylark()
+                headersWithoutPodSupport.toSkylark() .+. depHdrs.toSkylark()
```

Comment:
I think another option here is to do something similar to what I'm doing, but just add the `pod_support` glob into the main `headers.include` glob. I don't know if `glob` itself will complain about globs that match the same thing more than once or if it'll de-dupe internally before passing the targets/files on to Bazel. Also not sure if that's a preferred solution even if it does work.